dove/src/lib.rs

```diff
@@ -17,3 +17,5 @@ pub mod context;
 pub mod index;
 /// Dove configuration.
 pub mod manifest;
+/// for tests
+pub mod tests_helper;
```

Comment:
Shouldn't this module be `#[cfg(test)]`?

Reply:
Then the tests will not see this module